activesupport/test/core_ext/duration_test.rb

```diff
@@ -661,6 +661,30 @@ def test_durations_survive_yaml_serialization
     assert_equal 660, (d1 + 60).to_i
   end
 
+  def test_string_build_raises_error
+    string_build_response =
+      begin
+        ActiveSupport::Duration.build("9")
+      rescue StandardError => e
+        e
+      end
+
+    assert_kind_of TypeError, string_build_response
+    assert_equal string_build_response.message, "can't build an `ActiveSupport::Duration` from a `String`"
```

Comment:
These tests could be clearer and shorter with `assert_raises`:

```ruby
    error = assert_raises(TypeError) do
      ActiveSupport::Duration.build("9")
    end

    assert_equal "can't build an `ActiveSupport::Duration` from a `String`", error.message 
```

Reply:
@eugeneius: Updated in https://github.com/rails/rails/pull/37013/commits/f8dbaf04107371b56fc51b215b1fe0a8596218d0